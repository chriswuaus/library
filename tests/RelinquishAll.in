ReLINqUISH ALL 100000
ADD MEMBER BEANBABY
relinquish all 1001010
relinquish all 100000
member books 100000
add collection test.csv
rent 100000 111111
rent 100000 111112
rent 100000 111113
rent 100000 111114
rent 100000 111115
rent 100000 111116
rent 100000 111117
member books 100000
relinquish all 100000
member books 100000
exit
