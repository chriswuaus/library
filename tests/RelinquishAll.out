user: No members in system.

user: Success.

user: No such member in system.

user: Success.

user: Member not currently renting.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)

user: Success.

user: Member not currently renting.

user: Ending Library process.
